corehq/apps/reports/standard/cases/basic.py

```diff
@@ -112,8 +112,16 @@ def es_results(self):
                         raise BadRequestError()
             raise e
 
+    def get_deactivated_owner_ids(self, ):
```

Comment:
I don't think this is referenced anywhere

Reply:
https://github.com/dimagi/commcare-hq/pull/21564/commits/3a9a84cf6748338f07148342d8c0b3d5a8414ee2